pkg/infrastructure/kogito_images.go

```diff
@@ -32,6 +37,11 @@ func GetRuntimeImageVersion() string {
 	return getRuntimeImageVersion(version.Version)
 }
 
+// GetDefaultImage returns the default full image of the image name.
+func GetDefaultImage(imageName string) string {
```

Comment:
A nitpick, can you please move `GetDefaultImage` up so `GetRuntimeImageVersion()` and `getRuntimeImageVersion(v string)` are next to each other, as they are related?

Reply:
Well spotted!